week-4/Homework/mandatory/4-groceries.js

```diff
@@ -6,7 +6,14 @@
 let groceriesToBuy = [];
 
 let groceryList = {
```

Comment:
the idea of this exercise is to set the `groceryList` automatically by reading through `groceriesToBuy` array. I think what you have done here is to manually set the item properties.